requirements.txt

```diff
@@ -18,3 +18,6 @@ unidecode
 jieba
 piper_phonemize
 pypinyin
+
+gradio
+funasr
```

Comment:
We want to keep the requirements as simple as possible, as far as I know, fanasr is a huge package, would you mind changing to sherpa-onnx implementation (refer to https://k2-fsa.github.io/sherpa/onnx/sense-voice/python-api.html# )。 Also, I think users can intall gradio once they want to run this demo (you can document it), not necessary put it in requirements.txt.